arrow-site/docs/_data/sidebar-quickstart.yml

```
@@ -0,0 +1,22 @@
+options:
+  - title: Introduction
+    url: /quickstart/#rrow-is-a-library-for-typed-functional-programming-in-kotlin
+
+  - title: Setup
+    url: /quickstart/#setup
+
+  - title: Media
+    url: https://media.arrow-kt.io
+
+  - title: Training
+
+    nested_options:
+
+      - title: 47 Degrees
+        url: https://www.47deg.com/
+
+      - title: Instill
+        url: https://instil.co/
+
+  - title: License
```

```suggestion
  - title: Code of Conduct
    url: https://github.com/arrow-kt/arrow/blob/main/CODE_OF_CONDUCT.md

  - title: License
```

@israelperezglez , sorry, we already have Code of Conduct :rocket: